### ISSUE TITLE
Villager exploitable

### ISSUE THREAD

**Issue by johalun** [AppliedEnergistics/Applied-Energistics-2#6755]
### Describe the bug

First, thank you so much for adding a villager. That's awesome!

However, circular trades gives opportunity for exploit for de-zombified fluix researcher.

### How to reproduce the bug

De-zombify the fluix researcher

Now you can quadruple certus and fluix crystals

### Expected behavior

Would be better if "crystal -> emerald" and "emerald -> crystal" were both not in the trade table at the same time.

### Which minecraft version are you using?

1.19

### On which mod loaders does it happen?

Forge

### Crash log

n/a

**Comment by Technici4n** [comment]
Wouldn't there be the same issue with two villagers anyway? I think nerfing the recipes is a better fix... 😅

**Comment by johalun** [comment]
Yes, with multiple villagers you have to coordinate so the same thing doesn't happen.

**Comment by johalun** [comment]
I like being able to trade certus for emeralds but there are so many ways to trade farmable items for emeralds in vanilla villagers so I don't see that as important. Being able to buy fluix on the other hand is valuable since it's not easy renewable after the seeds were removed.
